rest-api/dao/questionnaire_response_dao.py

```diff
@@ -151,6 +151,11 @@ def _update_participant_summary(
     """
     # Block on other threads modifying the participant or participant summary.
     participant = ParticipantDao().get_for_update(session, questionnaire_response.participantId)
+
+    if participant is None:
```

Comment:
With this change, you should be able to remove the call to validate_participant_reference below in this method.

Reply:
The validate_participant_reference method is calling the validate_participant_id method

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/participant_dao.py#L157

which checks the withdrawn status, is this check not needed?

Reply:
That's already being checked here on line 178.

Reply:
OK, the test failure shows that we need to call raise_if_withdrawn not in an else block... it needs to happen in the insert case, too.